Add clarifications for the Bitbucket Server oAuth1 configuration (#2609)
codenvy/che-docs: modules/administration-guide/partials/proc_applying-an-application-link-secret-for-the-bitbucket-server.adoc

* The application link is set up on the Bitbucket Server.

* The following files, which were created when setting up the application link, are prepared:
** `privatepkcs8-stripped.pem`
** Base64-encoded `bitbucket-consumer-key`

> ** Base64-encoded `bitbucket-shared-secret`

Reviewer: This is because of my two suggestions below:
```suggestion
** `bitbucket-consumer-key`
** `bitbucket-shared-secret`
```

* The application link is set up on the Bitbucket Server.

* The following files, which were created when setting up the application link, are prepared:
** `privatepkcs8-stripped.pem`
** Base64-encoded `bitbucket-consumer-key`
** Base64-encoded `bitbucket-shared-secret`

> ** `privatepkcs8-stripped.pem`
> ** `bitbucket-consumer-key`
> ** `bitbucket-shared-secret`

Reviewer: Looks like some suggestion-caused extra lines:
```suggestion
```

* The application link is set up on the Bitbucket Server.

> * The following Base64-encoded values, which were generated when setting up the application link, are prepared:

Reviewer: ```suggestion
* The following files, which were created when setting up the application link, are prepared:
** `privatepkcs8-stripped.pem`
** Base64-encoded `bitbucket-consumer-key`
** Base64-encoded `bitbucket-shared-secret`
```

Author: I don't think this is relevant because we use the base64 encoded content but not the file itself

Author: However it is OK as `Prerequisites`

----
<1> The {prod-short} namespace. The default is `{prod-namespace}`.
<2> The URL of the Bitbucket Server.

> <3> The content of the `privatepkcs8-stripped.pem` file.

Reviewer: ```suggestion
<3> The content of the `privatepkcs8-stripped.pem` file, which was Base64-encoded when the file was generated.
```

> <3> The content of the `privatepkcs8-stripped.pem` file.
> <4> The Base64-encoded content of the `bitbucket-consumer-key` file.
> <5> The Base64-encoded content of the `bitbucket-shared-secret` file.

Reviewer: If this makes sense to you:
```suggestion
<3> The content of the `privatepkcs8-stripped.pem` file, which is already Base64-encoded.
<4> The content of the `bitbucket-consumer-key` file, which is already Base64-encoded.
<5> The content of the `bitbucket-shared-secret` file, which is already Base64-encoded.
```

Author: > <4> The content of the `bitbucket-consumer-key` file, which is already Base64-encoded.
> <5> The content of the `bitbucket-shared-secret` file, which is already Base64-encoded.

This is not relevant, The values are NOT base64 encoded in fact

> <4> The Base64-encoded content of the `bitbucket-consumer-key` file.
> <5> The Base64-encoded content of the `bitbucket-shared-secret` file.

Reviewer: ```suggestion
<4> The content of the `bitbucket-consumer-key` file, which you manually encoded to Base64.
<5> The content of the `bitbucket-shared-secret` file, which you manually encoded to Base64.
```

+
[IMPORTANT]
====
The content of the `privatepkcs8-stripped.pem` file is already Base64 encoded, so no need to run the `echoo <file-content> | base64` command, like for the `consumer-key` and `shared-secret` files.

> ====

Reviewer: If you decide to apply the two previous suggestions, then we can get rid of this IMPORTANT admonition:
```suggestion
```
